JustSaying.UnitTests/AwsTools/MessageHandling/SqsNotificationListener/WhenThereAreExceptionsInMessageProcessing.cs

```diff
@@ -52,10 +73,12 @@ protected override Task Given()
                     Arg.Any<ReceiveMessageRequest>(),
                     Arg.Any<CancellationToken>()))
                 .Do(x => _callCount++);
-            return Task.CompletedTask;
         }
 
-        protected override async Task When()
+
+#pragma warning disable CA1716
```

Comment:
What's this disable for?

Reply:
"When" is a reserved word. 
I was expecting to rename the method at some later PR (not sure whet to yet), in order to remove this `#pragma`

Reply:
Ah, I'd forgotten about `when` for exception clauses.